bundles/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/internal/NetatmoHandlerFactory.java

```diff
@@ -68,131 +48,52 @@
 @Component(service = ThingHandlerFactory.class, configurationPid = "binding.netatmo")
 public class NetatmoHandlerFactory extends BaseThingHandlerFactory {
     private final Logger logger = LoggerFactory.getLogger(NetatmoHandlerFactory.class);
-    private final Map<ThingUID, ServiceRegistration<?>> discoveryServiceRegs = new HashMap<>();
-    private final Map<ThingUID, ServiceRegistration<?>> webHookServiceRegs = new HashMap<>();
-    private final HttpService httpService;
-    private final NATherm1StateDescriptionProvider stateDescriptionProvider;
-    private final TimeZoneProvider timeZoneProvider;
-    private final LocaleProvider localeProvider;
-    private final TranslationProvider translationProvider;
-    private boolean backgroundDiscovery;
+
+    private final NetatmoDescriptionProvider stateDescriptionProvider;
+    private final ApiBridge apiBridge;
+    private final NetatmoServlet webhookServlet;
 
     @Activate
-    public NetatmoHandlerFactory(final @Reference HttpService httpService,
-            final @Reference NATherm1StateDescriptionProvider stateDescriptionProvider,
-            final @Reference TimeZoneProvider timeZoneProvider, final @Reference LocaleProvider localeProvider,
-            final @Reference TranslationProvider translationProvider) {
-        this.httpService = httpService;
+    public NetatmoHandlerFactory(@Reference HttpService httpService, @Reference ApiBridge apiBridge,
```

Comment:
You reference HttpService but you don't use it.
Is it necessary ?

Reply:
No, a forgotten leftover